esphome/components/scd4x/scd4x.cpp

```diff
@@ -150,6 +156,20 @@ void SCD4XComponent::update() {
     return;
   }
 
+  if (this->pressure_sensor_ != nullptr) {
+    float pressure = this->pressure_sensor_->state / 1000.0;
```

Comment:
We use single-precision floats in ESPHome:
```suggestion
    float pressure = this->pressure_sensor_->state / 1000.0f;
```